src/app/_models/question_submission.ts

```diff
@@ -17,6 +17,7 @@ export interface QuestionSubmission {
     no_file_answer: boolean;
     get_decoded_stderr: string;
     get_decoded_results: string[];
+    get_status_message: string;
```

Comment:
Please use an Enum as a type for this.
https://ce.judge0.com/#statuses-and-languages-status-get